dashboard/lib/metrics/events.rb

```diff
@@ -27,11 +27,12 @@ class << self
       # @event_value - the value of the event (optional)
       # @metadata - a metadata hash with relevant data (optional)
       # @get_enabled_experiments - include list of experiements the user is enrolled in (optional)
-      def log_event(user: nil, event_name:, event_value: nil, metadata: {}, get_enabled_experiments: false, session: nil)
+      # @cookies - the cookies hash (optional)
+      def log_event(user: nil, event_name:, event_value: nil, metadata: {}, get_enabled_experiments: false, cookies: nil)
```

Comment:
Since `cookies`, `session`, and `params` are all accessible through the `request`, it might make sense to pass only the `request`:
```suggestion
      def log_event(user: nil, event_name:, event_value: nil, metadata: {}, get_enabled_experiments: false, request: nil)
```

Reply:
That's a good idea. This will also clean things up in `sign_up_tracking.rb` where I'm passing in both session and cookies right now.